libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -207,14 +300,33 @@ class AP_Airspeed
         float last_saved_ratio;
         uint8_t counter;
     } state[AIRSPEED_MAX_SENSORS];
-
+    
+    struct blend_state_t {
+        float raw_airspeed;
+        float airspeed;
+        float last_pressure;
+        float filtered_pressure;
+        float corrected_pressure;
+        float EAS2TAS;
+        float temperature;
+        float ratio;
+        float offset;
+        
+        // Blended airspeed
+        AP_Int8 enable;
```

Comment:
ok, so, I was wrong. It's probably better to pull this param out as blend_enable and that way this function is safe to memset without a read modify write. Just write.

Reply:
Done